### ISSUE TITLE
[a11y] LIbrary

### ISSUE THREAD

**Issue by gregoriopellegrino** [readium/r2-testapp-swift#108]
When the user holds the book cover and on the screen appears the writing “Infos, Remove, Cancel”, the focus of VoiceOver doesn’t change thus making the buttons not accessible.

Tested on iOS 11.3 with VoiceOver
